content/en/apps/payments/payments-agent.md

```diff
@@ -0,0 +1,341 @@
+---
+date: '2020-01-08T09:59:25Z'
+menu:
+  apps:
+    parent: "payments"
+    name: The Payments Agent
+title: Learn more about the Payments Agent CorDapp
+weight: 300
+---
+
+The Payments Agent CorDapp is hosted by a trusted member of a network, such as the Business Network Operator (BNO). In this technical preview, you can assign the role of Payments Agent to a node on your local network.
+
+## Payments Agent and the PSP
+
+In the role of Payments Agent on a network, you are responsible for the connection between your node and the Payment Service Provider (PSP) you are using–in the technical preview, this is Modulr. You must have an account with Modulr in order to act as the Payments Agent, and to make use of this technical preview.
+
+## Customers of the Payments Agent
+
+As the Payments Agent, you can consider members of your network who wish to make payments as your **Customers**. In this documentation, any reference to a customer or customer ID refers to members of a network who wish to make payments using Corda Payments with you as their Payments Agent.
+
+Before you can create accounts for a party on your network, you must add them as a customer.
+
+Using the Payments Agent CorDapp, you can manage your customers' payment accounts and payment requests. You can use the Payments Agent CorDapp endpoints to connect to a UI to perform tasks as required.
+
+## Payments Agent flows
+
+The Payments Agent CorDapp provides flows that correspond to API endpoints. You can use these flows to:
+
+* Return a list of accounts based on defined search criteria. [Accounts](#accounts).
```

Comment:
What's the link on the end doing? Should that be part of the sentence?

Reply:
This is a long list of flow endpoints so the link links to the endpoint.